spec/routes.spec.ts

```diff
@@ -31,459 +31,474 @@ const testDocConfig = {
   servers: [{ url: 'v1' }],
 };
 
-describe('Routes', () => {
-  describe('response definitions', () => {
-    it('can set description', () => {
-      const registry = new OpenAPIRegistry();
-
-      registry.registerPath({
-        method: 'get',
-        path: '/',
-        responses: {
-          200: {
-            description: 'Simple response',
-            content: {
-              'application/json': {
-                schema: z.string(),
+describe.each`
+  type          | registerFunction     | rootDocPath
+  ${'Routes'}   | ${'registerPath'}    | ${'paths'}
+  ${'Webhooks'} | ${'registerWebhook'} | ${'webhooks'}
+`(
+  '$type',
+  ({
+    registerFunction,
+    rootDocPath,
+  }: {
+    registerFunction: 'registerPath' | 'registerWebhook';
+    rootDocPath: 'paths' | 'webhooks';
+  }) => {
```

Comment:
Can you please extract this function into a named one? That should make it a bit easier to read & hopefully there would be no indentation change needed and the diff would not be artificially big.